### ISSUE TITLE
[studio] Provide a log console in the global menu

### ISSUE THREAD

**Issue by avasquez614** [craftercms/craftercms#4050]
**Is your feature request related to a problem? Please describe.**
Right now when there's any type of failure in Studio (like a Groovy class compilation issue or a publishing problem), there's no way for a Studio administrator to check the logs without having access to the logs in the physical server. This normally means contacting the sysadmin just so he can read the logs, even if it is for a simple issue.

**Describe the solution you'd like**
We need to provide a log console in the global admin with 3 tabs:
- One that displays the `studio.log`
- One that displays the `engine.log` of preview
- One that displays the `deployer-general.log` of preview

These logs are already readable Studio (we can add the log paths to the configuration), so we can provide a log viewer API that receives the name of the log and the number of lines and returns the log lines to the UI, and then the UI will display this. The UI should also have a refresh button so that new lines can be retrieved or every X seconds request new lines and update the viewer.

**Comment by rart** [comment]
Which is the log currently on the log console? i.e.
<img width="500" alt="image" src="https://user-images.githubusercontent.com/3928341/81393487-bdf2ac00-9120-11ea-93d2-88426ea193e0.png">

This should be tagged with [studio-ui] too, no?

**Comment by sumerjabri** [comment]
@rart yes, it will flow over to the UI once the back is ready. I changed the title.

**Comment by rart** [comment]
Which is the log currently on the log console? i.e.
<img width="500" alt="image" src="https://user-images.githubusercontent.com/3928341/81393487-bdf2ac00-9120-11ea-93d2-88426ea193e0.png">

**Comment by jdrossl** [comment]
The studio log is already present in the global menu (that's the one that Roy is asking about), then each site already has it's own log in `Site Config`.

So the only one missing is the deployer.

**Comment by avasquez614** [comment]
@rart @jdrossl @sumerjabri the logs provided in that global console is very limited. If I compare the log entries the difference is pretty huge. The site log in the entries is even worse, I see maybe 2 entries while in the actual logs there are hundres. Right now those logs TBH are not useful.

I asked to use specific log files since those can be shown easily in the log console, and they are almost mirrors of what the standard outs of Tomcat and Deployer. I don't think we can do that in the site log because the log files have too much info.
